docs/standard/frameworks.md

```diff
@@ -14,139 +14,125 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
-| dnxcore50            |             |
-| dnx                  |             |
-| dnx45                |             |
-| dnx451               |             |
-| dnx452               |             |
-| dotnet               | netstandard |
-| dotnet50             |             |
-| dotnet51             |             |
-| dotnet52             |             |
-| dotnet53             |             |
-| dotnet54             |             |
-| dotnet55             |             |
-| dotnet56             |             |
-| netcore50            | uap10.0     |
-| win                  | netcore45   |
-| win8                 | netcore45   |
-| win81                | netcore451  |
-| win10                | uap10.0     |
-| winrt                | netcore45   |
-
-## Precedence
-
-A number of frameworks are related to and compatible with one another but not necessarily equivalent:
-
-| Framework                        | Can use   |
-| -------------------------------- | --------- |
-| uap (Universal Windows Platform) | win81     |
-|                                  | wpa81     |
-|                                  | netcore50 |
-| win (Windows Store)              | winrt     |
-|                                  | winrt45   |
-
-## .NET Standard
-
-The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For more information, see the [.NET Standard](net-standard.md) topic.
-
-The [NuGet Tools Get Nearest Framework Tool](http://nugettoolsdev.azurewebsites.net/) simulates the NuGet logic used for the selection of one framework from many available framework assets in a package based on the project's framework. To use the tool, enter one project framework and one or more package frameworks. Select the **Submit** button. The tool indicates if the package frameworks you list are compatible with the project framework you provide.
-
-## Portable Class Libraries
-
-For information on Portable Class Libraries, see the [Portable Class Libraries](/nuget/schema/target-frameworks#portable-class-libraries) section of the *Target Framework* topic in the NuGet documentation. Stephen Cleary created a tool that lists the supported PCLs. For more information, see [Framework Profiles in .NET](http://blog.stephencleary.com/2012/05/framework-profiles-in-net.html).
+When you target a framework in an app or library, you're specifying the set of APIs that you'd like to make available to the app or library. You specify the target framework in your *csproj* project file using Target Framework Monikers (TFMs).
+
+An app or library can target a version of [.NET Standard](~/docs/standard/net-standard.md). .NET Standard versions represent standardized sets of APIs across all .NET implementations. For example, a library can target .NET Standard 1.6 and gain access to APIs that function across .NET Core and .NET Framework using the same codebase.
```

Comment:
> .Net Standard applications were not supported

True, and I see what you mean that a reader could jump to the wrong conclusion. However, the statement isn't that `netstandardX.Y` apps exist; the statement is merely that "an app ... can target a version of .NET Standard," and that's a true statement ... even if *indirectly* true, since apps really "target" by .NET Core distribution and distributions (versions of `Microsoft.NETCore.App`) call for some flavor of `NETStandard.Library`.

The problem here is that this intro paragraph is making broad statements about target frameworks, and if we add a few sentences to try and explain further, the intro will probably get too heavy.

Another option is to split the "app" part out of this paragraph into its own paragraph. Then, we could say something slightly different about how .NET Core apps "target" with the `netcoreapp` target framework. I'm just not sure that would really help all that much at this point in the topic. The statement here is just meant to be a general remark pertaining to targeting the .NET Standard as opposed to other non-.NET Standard targeting.

I'm certainly open to suggestions. I just don't want to make this too long or too complicated.